package.json

```diff
@@ -18,8 +18,8 @@
     "geojson-vt": "^2.1.0",
     "gl-matrix": "https://github.com/toji/gl-matrix/archive/v2.2.1.tar.gz",
     "glify": "0.5.0",
-    "mapbox-gl-function": "^1.0.0",
-    "mapbox-gl-style-spec": "git+https://github.com/mapbox/mapbox-gl-style-spec.git#e9a261ed918ee8711f257a8d4acb5624daa48a24",
+    "mapbox-gl-function": "^2.0.0",
+    "mapbox-gl-style-spec": "git+https://github.com/mapbox/mapbox-gl-style-spec.git#v8",
```

Comment:
I used to do this (pin to a branch name), but I no longer think it's a good practice, because branch names are moving targets. Particularly `v8` -- what it resolves to today might work with this particular mapbox-gl-js commit, but as soon as someone commits additional spec changes, every historical commit will be broken. Modulo history rewrites, pinning to specific git hashes won't break in this way. And having pinned downstream dependencies is an excellent reason _not_ to rewrite history, so I'll stop doing that.